cms/static/js/models/group.js

```diff
@@ -20,14 +29,15 @@ define([
             return {
                 id: this.get('id'),
                 name: this.get('name'),
-                version: this.get('version')
+                version: this.get('version'),
+                usage: this.get('usage')
              };
         },
 
         validate: function(attrs) {
             if (!str.trim(attrs.name)) {
                 return {
-                    message: gettext('Group name is required'),
+                    message: gettext('Group name is required.'),
```

Comment:
In the context in which this is used, the period is not needed. I looked through all the other validate methods and none of them use periods. Please remove it.